package.json

```diff
@@ -15,27 +15,27 @@
     "test:unit:watch": "jest --watch src/tests/unit"
   },
   "dependencies": {
-    "@ckeditor/ckeditor5-alignment": "38.1.1",
-    "@ckeditor/ckeditor5-basic-styles": "38.1.1",
-    "@ckeditor/ckeditor5-block-quote": "38.1.1",
-    "@ckeditor/ckeditor5-build-decoupled-document": "^38.1.1",
-    "@ckeditor/ckeditor5-core": "38.1.1",
+    "@ckeditor/ckeditor5-alignment": "39.0.2",
+    "@ckeditor/ckeditor5-basic-styles": "39.0.2",
+    "@ckeditor/ckeditor5-block-quote": "39.0.2",
+    "@ckeditor/ckeditor5-build-decoupled-document": "39.0.2",
+    "@ckeditor/ckeditor5-core": "39.0.2",
     "@ckeditor/ckeditor5-dev-utils": "38.1.1",
```

Comment:
this has not updates to 39, it goes from v38 to v44.

Reply:
leave it. I think the dev utils do not follow the rest of the packages in terms of release/versioning